src/components/slider/slider-base.ts

```diff
@@ -544,8 +544,37 @@ export class IgcSliderBaseComponent extends LitElement {
     return groups;
   }
 
-  protected renderThumb(value: number, ariaLabel?: string, thumbId?: string) {
+  private handleFocus(ev: Event) {
+    this.activeThumb = ev.target as HTMLElement;
+    const thumbId = this.activeThumb?.id;
+    const thumbVal = this.activeThumb?.ariaValueNow;
+
+    const thumbs = this.shadowRoot?.querySelectorAll('div[part="thumb"]');
+    thumbs?.forEach((t) => {
+      if (t.id !== thumbId) {
+        this.activeThumb?.setAttribute(
+          'aria-valuetext',
+          `${Math.min(
+            parseInt(thumbVal!),
+            parseInt(t.ariaValueNow!)
+          )} - ${Math.max(parseInt(thumbVal!), parseInt(t.ariaValueNow!))}`
```

Comment:
Since step can be decimal, maybe parse to float and format it to step precision.
This should account for the **discrete** property as well, since when active the value is unbound from the step property.